e2e/v2-utils.ts

```diff
@@ -26,23 +48,11 @@ export async function selectAction(
     await ensureAssigned(page)
   }
 
-  // Keep retrying the click until the dropdown is visible
-  let isVisible = false
-  let attempts = 0
-  const maxAttempts = 10
-
-  while (!isVisible && attempts < maxAttempts) {
```

Comment:
This works fine without the while loop, tested here: https://github.com/opencrvs/e2e/actions/runs/18525107550